make-webpack-config.js

```diff
@@ -54,13 +54,12 @@ module.exports = function(rules, options) {
 
   if( specialOptions.separateStylesheets ) {
     plugins.push(new ExtractTextPlugin({
-      filename: '[name].css' + (specialOptions.longTermCaching ? '?[contenthash]' : ''),
+      filename: "[name].css" + (specialOptions.longTermCaching ? "?[contenthash]" : ""),
```

Comment:
One "downside" to my changes: `swagger-ui-bundle.css` and `swagger-ui-standalone-preset.css` files are now generated with the build. We could "fix" that by changing the `[name].css` to `swagger-ui.css`.